notebooks/DAAC_UWG_cloud_tutorial/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -0,0 +1,430 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b6ed7610",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<center>\n",
+    "<img src='./img/nsidc_logo.png'/>\n",
+    "    \n",
+    "# **Accessing and working with ICESat-2 data in the cloud**\n",
+    "    \n",
+    "</center>\n",
+    "    \n",
+    "\n",
+    "## **1. Tutorial Overview**\n",
+    "\n",
+    "**Note: This is an updated version of the notebook that was presented to the NSIDC User Working Group in May 2022**\n",
+    "\n",
+    "This notebook demonstrates searching for cloud-hosted ICESat-2 data and directly accessing Land Ice Height (ATL06) granules from an Amazon Compute Cloud (EC2) instance using the `earthaccess` package. NASA data \"in the cloud\" are stored in Amazon Web Services (AWS) Simple Storage Service (S3) Buckets. **Direct Access** is an efficient way to work with data stored in an S3 Bucket when you are working in the cloud. Cloud-hosted granules can be opened and loaded into memory without the need to download them first. This allows you take advantage of the scalability and power of cloud computing. \n",
+    "\n",
+    "The Amazon Global cloud is divided into geographical regions.  To have direct access to data stored in a region, our compute instance - the cloud server we are running our code on - must be in the same region as the data.  This is a fundamental concept of _analysis in place_. **NASA cloud-hosted data is in Amazon Region us-west2.  So your compute instance must also be in us-west2.** If we wanted to use data stored in another region, to use direct access for that data, we would start a compute instance in that region.\n",
+    "\n",
+    "As an example data  collection, we use ICESat-2 Land Ice Height (ATL06) over the Juneau Icefield, AK, for March 2003. ICESat-2 data granules, including ATL06, are stored in HDF5 format. We demonstrate how to open an HDF5 granule and access data variables using `xarray`. Land Ice Heights are then plotted using `hvplot`. \n",
+    "\n",
+    "`earthaccess` is a package developed by Luis Lopez (NSIDC developer) to allow easy search of NASA CMR and download of NASA data collection.  It can be used for programmatic search and access for both _DAAC-hosted_ and _cloud-hosted_ data. It manages authenticating using Earthdata Login credentials which are then used to obtain the S3 tokens that are needed for S3 direct access. https://github.com/nsidc/earthaccess\n",
```

Comment:
```suggestion
    "`earthaccess` is a package developed by Luis Lopez (NSIDC developer) to allow easy search of the NASA Common Metadata Repository (CMR) and download of NASA data collections.  It can be used for programmatic search and access for both _DAAC-hosted_ and _cloud-hosted_ data. It manages authenticating using Earthdata Login credentials which are then used to obtain the S3 tokens that are needed for S3 direct access. https://github.com/nsidc/earthaccess\n",
```